charts/sentry/values.yaml

```diff
@@ -1850,6 +1850,9 @@ mail:
 symbolicator:
   enabled: false
   api:
+    cleaner:
+      enabled: false
```

Comment:
Should we enable this per default ?

Reply:
@Mokto do I need to refactor the pull request?

Reply:
I would love to enable that per default for everyone if that's an issue.

Reply:
Review please